crates/bevy_ui/src/focus.rs

```diff
@@ -252,9 +252,13 @@ pub fn ui_focus_system(
             // The mouse position relative to the node
             // (0., 0.) is the top-left corner, (1., 1.) is the bottom-right corner
             // Coordinates are relative to the entire node, not just the visible region.
-            let relative_cursor_position = camera_cursor_positions
-                .get(&camera_entity)
-                .map(|cursor_position| (*cursor_position - node_rect.min) / node_rect.size());
+            let relative_cursor_position =
+                camera_cursor_positions
+                    .get(&camera_entity)
+                    .and_then(|cursor_position| {
+                        (node_rect.size().cmpgt(Vec2::ZERO).all())
```

Comment:
A comment on this line explaining why this is needed (maybe with a link to this PR) would be great.